<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>246</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>361</width>
    <height>246</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>361</width>
    <height>246</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>CRYPTO</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(15, 27, 64, 255), stop:1 rgba(1, 53, 106, 255));</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>271</x>
     <y>214</y>
     <width>87</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(13, 67, 105);;
border:1px solid rgb(28, 78, 115);
color: rgb(61, 173, 252);</string>
   </property>
   <property name="text">
    <string>ОК</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>1</y>
     <width>355</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(2, 47, 102);
border:1px solid #023d84;
color: rgb(19, 135, 250);

combobox-popup: 0;
selection-color:  #000e24;
selection-background-color: #0066ff;


</string>
   </property>
   <property name="maxVisibleItems">
    <number>3</number>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>56</y>
     <width>354</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(23, 41, 82, 255), stop:1 rgba(28, 51, 120, 255));
border:1px solid qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(29, 51, 92, 255), stop:1 rgba(36, 68, 145, 255));
/*1px solid red;
border: 1px solid rgb(45, 83, 155); */</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>214</y>
     <width>87</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(13, 67, 105);;
border:1px solid rgb(28, 78, 115);
color: rgb(61, 173, 252);</string>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>214</y>
     <width>87</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(13, 67, 105);;
border:1px solid rgb(28, 78, 115);
color: rgb(54, 152, 222);
color: rgb(61, 173, 252);</string>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>32</y>
     <width>352</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">/*color: rgb(28, 175, 138);
color: rgb(28, 217, 170);
color: rgb(127, 222, 194);
color:rgb(177, 244, 232);*/

color: rgb(24, 214, 186);
background: rgba(255,255,255,0);</string>
   </property>
   <property name="text">
    <string>0 BTC</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>62</y>
     <width>352</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(10, 141, 255);
background: rgba(255,255,255,0);</string>
   </property>
   <property name="text">
    <string>0 USD</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>91</y>
     <width>352</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(159, 41, 255);
background: rgba(255,255,255,0);
</string>
   </property>
   <property name="text">
    <string>0 RUB</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>86</y>
     <width>354</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(23, 41, 82, 255), stop:1 rgba(28, 51, 120, 255));
border:1px solid qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(29, 51, 92, 255), stop:1 rgba(36, 68, 145, 255));
/*1px solid red;
border: 1px solid rgb(45, 83, 155); */</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>115</y>
     <width>354</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(23, 41, 82, 255), stop:1 rgba(28, 51, 120, 255));
border:1px solid qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(29, 51, 92, 255), stop:1 rgba(36, 68, 145, 255));
/*1px solid red;
border: 1px solid rgb(45, 83, 155); */</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>145</y>
     <width>355</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(11, 67, 126);
border:1px solid rgb(130, 158, 189);
background-color: rgb(7, 48, 89);
color: rgb(20, 138, 255);
color: rgb(16, 184, 255);
/*#1067bd*/</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>180</y>
     <width>87</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid #073562;
background-color: rgb(7, 48, 89);
color: rgb(19, 135, 250);
color: rgb(16, 184, 255);
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>271</x>
     <y>180</y>
     <width>87</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid #073562;
background-color: rgb(7, 48, 89);
color: rgb(19, 135, 250);
color: rgb(16, 184, 255);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>180</y>
     <width>87</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid #073562;
background-color: rgb(7, 48, 89);
color: rgb(19, 135, 250);
color: rgb(16, 184, 255);</string>
   </property>
   <property name="text">
    <string>BTC</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_4">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>137</y>
     <width>354</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px ;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(5, 54, 107, 255), stop:1 rgba(8, 96, 190, 255));
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>121</y>
     <width>354</width>
     <height>13</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(0,0,0,0);
color: rgb(16, 66, 99);
color: rgb(21, 146, 255);</string>
   </property>
   <property name="text">
    <string>0%</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
